.github/workflows/bazel_build.yml

```diff
@@ -0,0 +1,21 @@
+name: Bazel Build
+run-name: ${{ github.actor }} is building with bazel
+on: [push]
+jobs:
+  bazel-build:
+    runs-on: ubuntu-22.04
+    steps:
+      - run: echo branch name is ${{ github.ref }}
+      - name: Checkout
+        uses: actions/checkout@v4.1.0
+      - name: Mount bazel cache
+        uses: actions/cache@v3
+        with:
+          path: "~/.cache/bazel"
+          key: "bazel-cache"
```

Comment:
I think this doesn't work as you're expecting.  Once a cache has been created, it will never be overwritten again (otherwise you could get races, when multiple actions are running).  Also see https://github.com/actions/toolkit/issues/505

Reply:
At this point it's hard to guess, what would be the good cache key.

~/.cache/bazel folder doesn't grow infinitely in my experience, the obsolete stuff is getting removed from there.

What maybe makes sense to add here a platform, once/if we would build for multiple platforms here.

Do you have ideas about good cache keys for this keys?

Reply:
Finally I got your point.

Indeed it works not as I was expecting.

Fixed from exampes: https://github.com/actions/cache/blob/main/examples.md#---bazel